src/basic/compress.c

```diff
@@ -234,15 +277,18 @@ int decompress_blob_lz4(
                 void **dst,
                 size_t* dst_size,
                 size_t dst_max) {
+        assert(src);
```

Comment:
Missing newline before this assert. (We always leave an empty line when a function declaration spans multiple lines)